### PR TITLE
Add GergoPlex and Faunchpad from gBoards

### PR DESCRIPTION
**Pull request by germ**
Adds in two new keyboards from gBoards, Faunchpad and GergoPlex. Also updates the Gergo readme

- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)

- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/faunchpad/rules.mk [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+MOUSEKEY_ENABLE 	= yes		# Mouse keys(+4700)
+EXTRAKEY_ENABLE 	= yes       # Audio control and System control(+450)
+CONSOLE_ENABLE 		= yes       # Console for debug(+400)
+COMMAND_ENABLE 		= no		# Commands for debug and configuration
+NKRO_ENABLE 		= yes       # USB Nkey Rollover
+STENO_ENABLE		= yes 		# Needed for chording
+
+OPT_DEFS += -DONLYQWERTY -DDEBUG_MATRIX
+SRC += sten.c
+EXTRAFLAGS += -flto
```
Comment (drashna) [inline keyboards/faunchpad/rules.mk]:
```suggestion
LTO_ENABLE = yes
```

File: keyboards/gergo/readme.md [hunk 2]
```diff
@@ -13,11 +13,11 @@ Hardware Availability: [gboards.ca](http://gboards.ca)
 ## Firmware building
 After cloning the QMK repo and installing dfu-programmer build and flash with. Be sure to reset your keyboard!
 
-    make gergo:germ:dfu
+    make gergo:default:dfu
```
Comment (drashna) [inline keyboards/gergo/readme.md]:
```suggestion
    make gergo:default:flash
```

File: keyboards/gergoplex/keymaps/default/config.h [hunk 3]
```diff
@@ -0,0 +1,2 @@
+#define COMBO_COUNT 16
```
Comment (drashna) [inline keyboards/gergoplex/keymaps/default/config.h]:
```suggestion
#pragma once 

#define COMBO_COUNT 16
```

File: keyboards/gergoplex/matrix.c [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+    debounce(raw_matrix, matrix, MATRIX_ROWS, changed);
+    matrix_scan_quantum();
+
+#ifdef DEBUG_MATRIX
+    for (uint8_t c = 0; c < MATRIX_COLS; c++)
+		for (uint8_t r = 0; r < MATRIX_ROWS; r++)
+		  if (matrix_is_on(r, c)) xprintf("r:%d c:%d \n", r, c);
+#endif
+
+    return 1;
```
Comment (drashna) [inline keyboards/gergoplex/matrix.c]:
```suggestion
    return changed;
```

File: keyboards/gergoplex/readme.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+A compact 30% (5x3+3) Split Keyboard from g Heavy Industries
+
+Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)  
+Hardware Supported: GergoPlex (Kit, Partial, Ready)   
+Hardware Availability: [gboards.ca](http://gboards.ca)  
+
+## Firmware building
+After cloning the QMK repo and installing dfu-programmer build and flash with. Be sure to reset your keyboard!
+
+    make gergoplex:default:dfu
```
Comment (drashna) [inline keyboards/gergoplex/readme.md]:
```suggestion
    make gergoplex:default:flash
```

File: keyboards/faunchpad/readme.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+Keyboard Maintainer: [Germ](https://github.com/germ)  
+Hardware Availability: [g Heavy Industries](https://www.gboards.ca/product)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make faunchpad:default
+
+Flash to your board with
+    make faunchpad:default:dfu
```
Comment (drashna) [inline keyboards/faunchpad/readme.md]:
```suggestion
    make faunchpad:default:flash
```

File: keyboards/faunchpad/readme.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# FaunchPad!
+
+![Faunchpad](https://assets.bigcartel.com/product_images/247416671/IMG_20191025_1322045-01.jpeg)
+
+A macropad loaded with a numpad, and a ton of cat macros. Spam some keys and push enter for cats, What more could you want from a keyboard?
+
+Keyboard Maintainer: [Germ](https://github.com/germ)
```
Comment (drashna) [inline keyboards/faunchpad/readme.md]:
```suggestion
* Keyboard Maintainer: [Germ](https://github.com/germ)  
```

File: keyboards/faunchpad/readme.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+# FaunchPad!
+
+![Faunchpad](https://assets.bigcartel.com/product_images/247416671/IMG_20191025_1322045-01.jpeg)
+
+A macropad loaded with a numpad, and a ton of cat macros. Spam some keys and push enter for cats, What more could you want from a keyboard?
+
+Keyboard Maintainer: [Germ](https://github.com/germ)  
+Hardware Availability: [g Heavy Industries](https://www.gboards.ca/product)
```
Comment (drashna) [inline keyboards/faunchpad/readme.md]:
```suggestion
* Hardware Availability: [g Heavy Industries](https://www.gboards.ca/product)
```

File: keyboards/gergoplex/keymaps/default/rules.mk [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+#Debug options
+VERBOSE 		 							= yes
+DEBUG_MATRIX_SCAN_RATE   	= no
+DEBUG_MATRIX		 					= yes
+
+# A bunch of stuff that you shouldn't touch unless you
+# know what you're doing.
+#
+# No touchy, capiche?
+SRC += matrix.c i2c_master.c
```
Comment (drashna) [inline keyboards/gergoplex/keymaps/default/rules.mk]: I don't think this needs/should be here.

```suggestion
```

File: keyboards/gergoplex/readme.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
+# GergoPlex
+
+![GergoPlex](https://assets.bigcartel.com/product_images/248890490/IMG_20191114_1406385-01-01.jpeg)
+
+A compact 30% (5x3+3) Split Keyboard from g Heavy Industries
+
+Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)
```
Comment (drashna) [inline keyboards/gergoplex/readme.md]:
```suggestion
* Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)  
```

File: keyboards/gergoplex/readme.md [hunk 11]
```diff
@@ -0,0 +1,22 @@
+# GergoPlex
+
+![GergoPlex](https://assets.bigcartel.com/product_images/248890490/IMG_20191114_1406385-01-01.jpeg)
+
+A compact 30% (5x3+3) Split Keyboard from g Heavy Industries
+
+Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)  
+Hardware Supported: GergoPlex (Kit, Partial, Ready)
```
Comment (drashna) [inline keyboards/gergoplex/readme.md]:
```suggestion
* Hardware Supported: GergoPlex (Kit, Partial, Ready)   
```

File: keyboards/gergoplex/readme.md [hunk 12]
```diff
@@ -0,0 +1,22 @@
+# GergoPlex
+
+![GergoPlex](https://assets.bigcartel.com/product_images/248890490/IMG_20191114_1406385-01-01.jpeg)
+
+A compact 30% (5x3+3) Split Keyboard from g Heavy Industries
+
+Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)  
+Hardware Supported: GergoPlex (Kit, Partial, Ready)   
+Hardware Availability: [gboards.ca](http://gboards.ca)
```
Comment (drashna) [inline keyboards/gergoplex/readme.md]:
```suggestion
* Hardware Availability: [gboards.ca](http://gboards.ca)  
```

**Comment by germ** [comment]
Thanks drash :)

**Comment by germ** [comment]
@zvecr give me a few and I'll get that done

**Comment by germ** [comment]
Any changes that still need to be done here?

File: keyboards/faunchpad/config.h [hunk 13]
```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x1337
+#define DEVICE_VER      1
+#define MANUFACTURER    g Heavy Industries
+#define PRODUCT     		FaunchPad
```
Comment (fauxpark) [inline keyboards/faunchpad/config.h]: Would you mind fixing all the mixed tabs/spaces in these files? :)

File: keyboards/faunchpad/rules.mk [hunk 14]
```diff
@@ -0,0 +1,23 @@
...
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+
+MOUSEKEY_ENABLE 	= yes		# Mouse keys(+4700)
```
Comment (fauxpark) [inline keyboards/faunchpad/rules.mk]:
```suggestion
MOUSEKEY_ENABLE 	= yes		# Mouse keys
```

File: keyboards/faunchpad/rules.mk [hunk 15]
```diff
@@ -0,0 +1,23 @@
...
+EXTRAKEY_ENABLE 	= yes       # Audio control and System control(+450)
```
Comment (fauxpark) [inline keyboards/faunchpad/rules.mk]:
```suggestion
EXTRAKEY_ENABLE 	= yes       # Audio control and System control
```

File: keyboards/faunchpad/rules.mk [hunk 16]
```diff
@@ -0,0 +1,23 @@
...
+CONSOLE_ENABLE 		= yes       # Console for debug(+400)
```
Comment (fauxpark) [inline keyboards/faunchpad/rules.mk]:
```suggestion
CONSOLE_ENABLE 		= yes       # Console for debug
```

File: keyboards/gergoplex/keymaps/default/rules.mk [hunk 17]
```diff
@@ -0,0 +1,19 @@
+#----------------------------------------------------------------------------
+# make gergoplex:default:flash
+# Make sure you have dfu-programmer installed!
+#----------------------------------------------------------------------------
+# Firmware options
+MOUSEKEY_ENABLE = yes  		# Mouse keys(+4700), needed for baller
```
Comment (fauxpark) [inline keyboards/gergoplex/keymaps/default/rules.mk]:
```suggestion
MOUSEKEY_ENABLE = yes  		# Mouse keys, needed for baller
```

File: keyboards/gergoplex/readme.md [hunk 18]
```diff
@@ -0,0 +1,22 @@
+# GergoPlex
+
+![GergoPlex](https://assets.bigcartel.com/product_images/248890490/IMG_20191114_1406385-01-01.jpeg)
+
+A compact 30% (5x3+3) Split Keyboard from g Heavy Industries
+
+* Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)
```
Comment (fauxpark) [inline keyboards/gergoplex/readme.md]:
```suggestion
* Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)
```

File: keyboards/gergoplex/readme.md [hunk 19]
```diff
@@ -0,0 +1,22 @@
+# GergoPlex
+
+![GergoPlex](https://assets.bigcartel.com/product_images/248890490/IMG_20191114_1406385-01-01.jpeg)
+
+A compact 30% (5x3+3) Split Keyboard from g Heavy Industries
+
+* Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)  
+* Hardware Supported: GergoPlex (Kit, Partial, Ready)
```
Comment (fauxpark) [inline keyboards/gergoplex/readme.md]:
```suggestion
* Hardware Supported: GergoPlex (Kit, Partial, Ready)
```

File: keyboards/gergoplex/readme.md [hunk 20]
```diff
@@ -0,0 +1,22 @@
+# GergoPlex
+
+![GergoPlex](https://assets.bigcartel.com/product_images/248890490/IMG_20191114_1406385-01-01.jpeg)
+
+A compact 30% (5x3+3) Split Keyboard from g Heavy Industries
+
+* Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)  
+* Hardware Supported: GergoPlex (Kit, Partial, Ready)   
+* Hardware Availability: [gboards.ca](http://gboards.ca)
```
Comment (fauxpark) [inline keyboards/gergoplex/readme.md]:
```suggestion
* Hardware Availability: [gboards.ca](http://gboards.ca)
```

File: keyboards/faunchpad/config.h (hunk 13 above)
Comment (germ) [reply keyboards/faunchpad/config.h]: Will do in next commit

Comment (germ) [reply keyboards/faunchpad/config.h]: Fixed in c6d5906

File: keyboards/faunchpad/config.h [hunk 21]
```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x1337
+#define DEVICE_VER      1
+#define MANUFACTURER    g Heavy Industries
+#define PRODUCT       	FaunchPad
```
Comment (drashna) [inline keyboards/faunchpad/config.h]:
```suggestion
#define PRODUCT         FaunchPad
```

File: keyboards/faunchpad/config.h [hunk 22]
```diff
@@ -0,0 +1,23 @@
...
+#define DESCRIPTION     Teeny Toiny Macropad
+#define VERSION   			"Rev 1"
```
Comment (drashna) [inline keyboards/faunchpad/config.h]:
```suggestion
#define VERSION          "Rev 1"
```

**Comment by germ** [comment]
Should be good to go <3

**Comment by germ** [comment]
We good to merge into master?

File: keyboards/gergo/readme.md [hunk 23]
```diff
@@ -13,11 +13,11 @@ Hardware Availability: [gboards.ca](http://gboards.ca)
 ## Firmware building
 After cloning the QMK repo and installing dfu-programmer build and flash with. Be sure to reset your keyboard!
 
-    make gergo:germ:dfu
+    make gergo:default:flash
 
 To just test your build with the default keymap
    
-    make gergo:germ
+    make default:germ
```
Comment (zvecr) [inline keyboards/gergo/readme.md]:
```suggestion
    make gergo:default
```

File: keyboards/gergoplex/config.h [hunk 24]
```diff
@@ -0,0 +1,59 @@
...
+#define MOUSEKEY_INTERVAL       20
+#define MOUSEKEY_DELAY          0
+#define MOUSEKEY_TIME_TO_MAX    60
+#define MOUSEKEY_MAX_SPEED      7
+#define MOUSEKEY_WHEEL_DELAY 0
+#define TAPPING_TOGGLE  1
+#define TAPPING_TERM    200
+#define IGNORE_MOD_TAP_INTERRUPT 
+#define LOCKING_SUPPORT_ENABLE
+#define LOCKING_RESYNC_ENABLE
```
Comment (zvecr) [inline keyboards/gergoplex/config.h]: Not a massive fan of all these overrides of defaults being at the keyboard level.

Comment (germ) [reply keyboards/gergoplex/config.h]: Worst case #undef !

Comment (drashna) [reply keyboards/gergoplex/config.h]: I do have to agree with the tapping term and IMTI settings defined here.  

Tapping term is defined as the default, so there shouldn't be a need to define it here, and you're forcing it to be undefined.

Comment (germ) [reply keyboards/gergoplex/config.h]: moved to layout config.h in 95a3b84

**Comment by germ** [comment]
Closing due to #8262

**Comment by germ** [comment]
G

**Pull request closed without merging**